crates/wasmtime/src/runtime/vm/instance/allocator/wasmfx_allocator.rs

```diff
@@ -0,0 +1,261 @@
+// WasmFX fiber stack allocators
+//
+// * on_demand: allocates memory lazily
+// * pooling: preallocates a chunk of memory eagerly
+//
+
+use anyhow::Result;
+use wasmtime_continuations::WasmFXConfig;
+
+cfg_if::cfg_if! {
```

Comment:
How about making each wasmfx implementation export the `Fiber` type it uses?

Reply:
Yeah, we should do something like that.